src/main/java/com/example/fooddelivery/model/Address.java

```diff
@@ -0,0 +1,38 @@
+package com.example.fooddelivery.model;
+
+import java.util.Set;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.validation.constraints.NotNull;
+
+@Entity(name = "addresses")
+public class Address {
+
+    @Id
+    @GeneratedValue
+    @Column
+    private Long id;
+
+    @NotNull
+    @Column
+    private String city;
+
+    @NotNull
+    @Column
+    private String street;
+
+    @NotNull
+    @Column
+    private Integer number;
+
+    @OneToMany(mappedBy = "address", cascade = CascadeType.ALL)
+    private Set<User> users;
```

Comment:
Use cascade from Parent to Child. In this way we avoid a user delete when an address delete is triggered.

Reply:
I did this